lib/fastlane/plugin/test_center/helper/multi_scan_manager/retrying_scan_helper.rb

```diff
@@ -98,11 +167,14 @@ def scan_options
             FastlaneCore::UI.important('Disabling -quiet as failing tests cannot be found with it enabled.')
             xcargs.gsub!('-quiet', '')
           end
-          xcargs.gsub!(/-parallel-testing-enabled(=|\s+)(YES|NO)/, '')
+          if FastlaneCore::Helper.xcode_at_least?(10)
```

Comment:
@lyndsey-ferguson this has been left out from the 3.15.0 release

Reply:
😱  Sorry

Reply:
I managed to solve it by adding this logic to the `override_scan_options_block` but it's probably better to have this fix in the plugin.

Reply:
Would you be able to do me the favor of opening a PR?  I don't have a lot of time -- but could possibly get to it this weekend.